mantidproject/mantid
# Release Notes 4.2 Step3

**Description of work.**

Resized images, changed ``*`` to \* to input asterisk correctly, Deleted ui.rst as unnecessary.

**To test:**

Build docs.html and check links and images are as expected.

Fixes #27307 

ref #27100 

*This does not require release notes* because **It is release notes!**

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
